pages/help/help_domain-requests.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Domain requests
+permalink: /help/domain-requests/
+layout: layouts/info-page
+searchbox: true
+excerpt: Learn how to request a .gov domain.
+tags: help
+eleventyNavigation:
+  key: help
+  order: 1 
```

Comment:
Totally fine with this being `1` but then `help_index` will need to be `order: 0`